addons/web/static/src/core/user_service.js

```diff
@@ -55,15 +55,15 @@ export const userService = {
             },
             get db() {
                 const res = {
-                    name: sessionInfo.db,
```

Comment:
Is this a new required convention? The commit is kinda polluted with those changes and I wonder if your editor fixed them all by mistake or something (the opposite convention was decided a long time ago :grimacing:)

Reply:
(see above comment about us now running prettier on all files that have been touched in web in pre-commit hook)